Fix language and typos (#979)
grafana/grafana-infinity-datasource: docs/sources/xml.md

{{< docs/play title="Infinity plugin XML demo" url="https://play.grafana.org/d/infinity-xml" >}}

> In the example below, we are going to use XML response in Grafana

Reviewer: ```suggestion
The following example, uses an XML response in Grafana:
```

</users>
```

> In the XML example above, we want to list the user's name and age. To do so, use the syntax below:

Reviewer: ```suggestion
If you want to get the user's name and age from the preceding XML example, use the following syntax:
```

</users>
```

> In the above example XML, we want to list the user's name and age. To do so, use the syntax below:

Reviewer: ```suggestion
If you want to get the user's name and age from the preceding XML example, use the following syntax:
```

Rows/Root: `users.user` to select all the user objects inside user element.

**Columns:**

Column1 Name: Use `_` as the selector for the property `name`. Here `_` refers to the text inside the node.

Column2 Age: Use `$.age` as the selector for the attribute `age`.

## Example 3

> Consider the example below:

Reviewer: ```suggestion
Consider the following example:
```

  </users>
```

> In the XML eample aboce, we want to list the user's name. To do so, use the syntax below:

Reviewer: ```suggestion
If you want to get the user's name and age from the preceding XML example, use the following syntax:
```

Rows/Root: `users.user` to select all the user objects inside user element

**Columns:**

Column1 Name: Use `_` as the selector for the property `name`. Here `_` refers to the text inside the node

## XML URL

> You can also use the hosted XML via the URL. The example shown below demonstrates how we connect XML from this [gist](https://gist.githubusercontent.com/yesoreyeram/655a362eed0f51be24e16d3f1127a31d/raw/aa58549a5cf9d06dae1204b5a09be5d651adc744/text.xml).

Reviewer: ```suggestion
You can also use the hosted XML via the URL. The following example shows how you connect XML using [a GitHub gist](https://gist.githubusercontent.com/yesoreyeram/655a362eed0f51be24e16d3f1127a31d/raw/aa58549a5cf9d06dae1204b5a09be5d651adc744/text.xml).
```

![image](https://user-images.githubusercontent.com/153843/99293208-39ed4f80-283a-11eb-831e-ae14d297a2f3.png#center)

> The image below shows a sample example visualized using bar gauge panel and time series format:

Reviewer: ```suggestion
The following image shows a sample example visualized using bar gauge panel and time series format:
```
